tests/unit/conftest.py

```diff
@@ -332,7 +332,6 @@ def full_record():
             'publisher': 'Jol'
         },
         communities=['zenodo'],
-        provisional_communities=['ecfunded'],
```

Comment:
Because of ``"additionalParameters": false`` this is no longer allowed, but I guess it's only for ES?